GTG/core/timer.py

```diff
@@ -70,15 +75,55 @@ def time_changed(self):
         self.timeout_source = GObject.timeout_add_seconds(secs_to_refresh,
                                                           self.emit_refresh)
 
-    def set_configuration(self, refresh_hour, refresh_min):
+    def set_configuration(self, time):
         try:
-            datetime.time(int(refresh_hour), int(refresh_min))
+            time
         except (ValueError, TypeError):
-            raise ValueError("{}:{} is not a valid time".format(refresh_hour,
-                                                                refresh_min))
+            raise ValueError("{} is not a valid time".format(time))
 
-        self.config.set('hour', refresh_hour)
-        self.config.set('min', refresh_min)
+        self.config.set('hour', time.hour)
+        self.config.set('min', time.minute)
 
     def get_configuration(self):
-        return self.config.get('hour'), self.config.get('min')
+        return datetime.time(int(self.config.get('hour')),
+                             int(self.config.get('min')))
```

Comment:
Put this inside try: except block and when ValueError is raised, log the error and return the default configuration instead `datetime.time(0, 0)`